### PR TITLE
Fix subs leak and fix schema error

### PR DESCRIPTION
**Pull request by ibrkhalil**
Fixes a subs leak and a schema error

### CODE REVIEW

File: src/legacy/status_im/subs/root.cljs [hunk 1]
```diff
@@ -22,7 +22,6 @@
 (reg-root-key-sub :peers-count :peers-count)
 (reg-root-key-sub :peers-summary :peers-summary)
 (reg-root-key-sub :web3-node-version :web3-node-version)
-(reg-root-key-sub :alert-banners :alert-banners)
```
Comment (flexsurfer) [inline src/legacy/status_im/subs/root.cljs:25]: could you elaborate?

**Review by flexsurfer — APPROVED** [review]

**Comment by flexsurfer** [comment]
seems like Schema is broken, take a look on tests

File: src/legacy/status_im/subs/root.cljs (hunk 1 above)
Comment (ibrkhalil) [reply src/legacy/status_im/subs/root.cljs:25]: It already exists here
https://github.com/status-im/status-mobile/blob/1129a9c3cbb299814506660eee7c53567bbc3073/src/status_im/subs/root.cljs#L67

**Comment by ibrkhalil** [comment]
E2E test failed not related to current PR
Merging

**Pull request merged**